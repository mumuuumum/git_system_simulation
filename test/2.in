3
write file1 1 10
BlueSharks
write file2 3 10
BlueSharks
ls