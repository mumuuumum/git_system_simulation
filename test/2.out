2 file1 file2
